server/src/test/java/org/elasticsearch/search/aggregations/metrics/SumIT.java

```diff
@@ -111,20 +107,9 @@ public void testEmptyAggregation() throws Exception {
         assertThat(sum.getValue(), equalTo(0.0));
     }
 
+    /** This test has been moved to {@link SumAggregatorTests#testUnmapped()} */
```

Comment:
These are because this class still extends AbstractNumericTestCase - the remainder of the tests will migrate in another pr